scripts/scripts.js

```diff
@@ -70,6 +70,10 @@ async function loadEager(doc) {
     document.body.classList.add('appear');
     await waitForLCP(LCP_BLOCKS);
   }
+  if (window.innerWidth >= 900) loadCSS(`${window.hlx.codeBasePath}/styles/fonts.css`);
+  if (sessionStorage.getItem('fonts-loaded')) {
+    loadCSS(`${window.hlx.codeBasePath}/styles/fonts.css`);
+  }
```

Comment:
This is calling `loadCSS` twice on desktop the 2nd time... browser is probably smart but could be optimised as:

```suggestion
  if (window.innerWidth >= 900 || sessionStorage.getItem('fonts-loaded')) {
    // load fonts.css immediately on desktop or if font has already been loaded
    loadCSS(`${window.hlx.codeBasePath}/styles/fonts.css`);
  }
```

Reply:
i am not sure the extra code is needed given that `loadCSS()` already checks